keyboards/preonic/keymaps/isouk/keymap.c

```diff
@@ -0,0 +1,385 @@
+     * `-----------------------------------------------------------------------------------'
+     */
+[_RAISE] = LAYOUT_preonic_grid( \
+        RELOAD,        BEST,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_SLSH,       LSFT(KC_8),    KC_LBRC, KC_RBRC, KC_MINS, \
+        KC_GRV,        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_NO,   KC_NO,   LCTL(KC_MINS), LCTL(KC_PLUS), KC_NO,   KC_NO,   KC_INS, \
+        KC_NLCK,       KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_NO,   KC_NO,   KC_MINS,       KC_PLUS,       KC_NO,   KC_NO,   KC_BSLS, \
+        LSFT(KC_NUBS), KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NO,   KC_NO,   KC_NO,         KC_PGUP,       KC_PGDN, KC_NO,   KC_NUHS, \
+        KC_NO,         KC_NO,   KC_NO,   KC_LGUI, KC_TRNS, KC_NO,   KC_NO,   KC_TRNS,       KC_MNXT,       KC_VOLD, KC_VOLU, KC_MPLY \
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Reset|      |      |      |      |      |      |      |      |      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|AudOff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|MusOff|MidiOn|MidOff|      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_preonic_grid( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+  _______, RESET,   DEBUG,   _______, _______, _______, _______, TERM_ON, TERM_OFF,_______, _______, KC_DEL,  \
+  _______, _______, MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, QWERTY,  COLEMAK, DVORAK,  _______, _______, \
+  _______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  \
+)
+
+
+};
+
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+  switch (keycode) {
+      case SECTION:
+        if (record->event.pressed) {
+            tap_code16(LCTL(LSFT(KC_U)));
+            wait_ms(10);
+            SEND_STRING("00a7");
+            wait_ms(10);
+            tap_code(KC_ENT);
+      }
+            return false;
+         break;
+      case BEST:
+        if (record->event.pressed) {
+            tap_code16(LALT(KC_D));
```

Comment:
`SEND_STRING_DELAY` will work much better for this.  It adds the delays between the key presses.

Reply:
yes, but this works already and has no bugs also i don't have the skill to implement new commands. it's in a rarely run macro that's not going to slow down the keyboard.